resources/views/components/checkbox.blade.php

```diff
@@ -1,31 +1,45 @@
 <div>
-    <label for="{{ $id }}" class="flex items-center {{ $errors->has($name) ? 'text-negative-600':'' }}">
-        @if ($leftLabel)
-            <x-dynamic-component
-                :component="WireUi::component('label')"
-                class="mr-2"
-                :for="$id"
-                :label="$leftLabel"
-                :has-error="$errors->has($name)"
-            />
-        @endif
+        <label for="{{ $id }}" class="flex items-center {{ $errors->has($name) ? 'text-negative-600':'' }}">
+            <div class="relative flex items-start">
+                @if ($leftLabel)
+                    <div class="mr-2 text-sm text-right">
+                        <x-dynamic-component
+                            :component="WireUi::component('label')"
+                            class=""
+                            :for="$id"
+                            :label="$leftLabel"
+                            :has-error="$errors->has($name)"
+                        />
+                        @if($description)
+                            <div id="{{ $id }} . comments-description" class="text-gray-500">{{ $description }}</div>
```

Comment:
Remove the id, it's not useful here
```suggestion
                            <div class="text-gray-500">{{ $description }}</div>
```